codegen/src/main/kotlin/software/amazon/smithy/rust/codegen/rustlang/CargoDependency.kt

```diff
@@ -192,6 +192,7 @@ data class CargoDependency(
         val Axum: CargoDependency = CargoDependency("axum", CratesIo("0.3"))
         val Bytes: CargoDependency = CargoDependency("bytes", CratesIo("1"))
         val BytesUtils: CargoDependency = CargoDependency("bytes-utils", CratesIo("0.1.1"))
+        val DeriveBuilder = CargoDependency("derive_builder", CratesIo("0.10"))
```

Comment:
in the long term, you _probably_ just want to code generate builders directly, I suspect. `syn` is a very large dependency that ideally, won't be a dependency of the generated code

Reply:
Good point. https://github.com/awslabs/smithy-rs/issues/864